### DISCUSSION TITLE
Github copilot - Errors founds

### DISCUSSION THREAD

**Discussion by jasp402** [community/community#26750] [Actions]
<p>Hi, community. I have been working with GitHub Copilot for about 3 weeks now and there is an error that I see a lot of recurring. I will try to explain it as detailed as possible.</p>
<p>The autocomplete has worked very well for me, however when I try to autocomplete an IP address it is as if something happened to it and it leaves the syntax incomplete.</p>
<pre data-code-wrap="js"><code class="lang-nohighlight">//create a variable with ipAddress fake
let ipAddress = "
</code></pre>
<p>The same happens when the word ip Address appears in another context</p>
<pre data-code-wrap="js"><code class="lang-nohighlight"> "keys"  :[
          "userslug",
          "location",
          "device",
          "ipAddress",
          "status"
        ], 
        "values":[
          "O5Pi0XwMd1buYIal",
          "LIMA",
          "Verifone",
          "
</code></pre>
<p>notice that it does not complete the quotes, nor does it close the array.</p>
<blockquote>
<p>Finally I want to apologize for my bad English. I hope this info is useful to someone and we compare knowledge about it.</p>
</blockquote>

**Comment by nethgato** [comment, marked as answer]
<p>Hey there <a class="mention" href="/jasp402">@jasp402</a> :wave:</p>
<p>First, please do not apologize for your English, you’re a-okay =)</p>
<p>Though, we’ve been gathering Copilot feedback, here:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback/discussions/categories/copilot-feedback">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback/discussions/categories/copilot-feedback" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/62281706/181099740-d845073a-02f0-4d5f-ad11-ab3018856b21.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/f/6f6575e1762df495b08e83d0afcfccd67930dcf3_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/f/6f6575e1762df495b08e83d0afcfccd67930dcf3_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/f/6f6575e1762df495b08e83d0afcfccd67930dcf3.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/f/6f6575e1762df495b08e83d0afcfccd67930dcf3_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback/discussions/categories/copilot-feedback" target="_blank" rel="noopener">Discussions · github/feedback</a></h3>

  <p>Public feedback discussions for: GitHub Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and general product feedback. - Discussions · github/feedback</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>And if you haven’t already, I would suggest submitting this in that space.</p>
<p>Be well!</p>
